openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -188,10 +192,25 @@ def _create_jwt(
     return _encode_and_sign(payload, use_asymmetric_key, secret)
 
 
+# .. toggle_name: FORCE_USE_ASYMMETRIC_KEY
```

Comment:
Could we use a name that is more clearly related to JWTs, and to their creation rather than verification?